app/controllers/internal/organizations_controller.rb

```diff
@@ -0,0 +1,18 @@
+class Internal::OrganizationsController < Internal::ApplicationController
+  layout "internal"
+
+  def index
+    @organizations = Organization.order("name DESC")
+
+    return if params[:search].blank?
+
+    @organizations = Organization.where(
```

Comment:
when `params[:search]` is not blank, it looks like line 5 will also get executed, even though it's going to be used

Reply:
Nice catch! I forgot to put the sigil there :grimacing: